mlir/utils/vscode/src/mlirContext.ts

```diff
@@ -179,14 +179,19 @@ export class MLIRContext implements vscode.Disposable {
 
     // Initialize additional configurations for this server.
     if (languageName === 'pdll') {
+      additionalServerArgs = config.get<string[]>("pdll_additional_server_args", null, []);
       await this.preparePDLLServerOptions(workspaceFolder, configsToWatch,
                                           filepathsToWatch,
                                           additionalServerArgs);
     } else if (languageName == 'tablegen') {
+      additionalServerArgs = config.get<string[]>("table_additional_server_args", null, []);
```

Comment:
```suggestion
      additionalServerArgs = config.get<string[]>("tablegen_additional_server_args", null, []);
```

Reply:
Can you just pull out the additional argument initialization to the set definition and generalize with `languageName + "_additional_server_args"`

Reply:
Fixed in https://github.com/llvm/llvm-project/pull/79671/commits/fe3d7e5f8f1a0821a98199430ebaa572bf59289a